src/Nest/Analysis/Tokenizers/NoriTokenizer.cs

```diff
@@ -47,6 +53,13 @@ public interface INoriTokenizer : ITokenizer
 		/// </remarks>
 		[DataMember(Name = "user_dictionary_rules")]
 		IEnumerable<string> UserDictionaryRules { get; set; }
+
+		/// <summary>
+		/// Filters to apply to this tokenizer.
+		/// For example:  "nori_number" - token filter normalizes Korean numbers to regular Arabic decimal numbers in half-width characters.
+		/// </summary>
+		[DataMember(Name = "filter")]
+		IEnumerable<string> Filters { get; set; }
```

Comment:
I think this should be removed. The token filters are applied at the `analyzer` level (level above `tokenizer`)

Reply:
++ thanks, please check https://github.com/elastic/elasticsearch-net/pull/4591/commits/0cb682b14bf1869fc29de3fec57c37531102ed30